steward/src/allocation.rs

```diff
@@ -253,7 +254,10 @@ pub async fn direct_rebalance(
     let mut contract_state = UniswapV3CellarState::new(cellar_address, client);
     for tick_weight in tick_weight {
         if tick_weight.weight > 0 {
-            tick_info.push(from_tick_weight(tick_weight.clone()))
+            tick_info.push(from_tick_weight(
+                tick_weight.clone(),
+                config.cellars[0].clone(),
```

Comment:
This technically works when there is only one cellar, but what we want is the cellar config whose cellar address matches the address from the request.